composer.json

```
@@ -7,7 +7,7 @@
 		"test": "vendor/bin/phpstan analyse -c tests/phpstan-config.neon --error-format oneline --no-progress --no-interaction"
 	},
     "require": {
-        "php": "^7.1",
+        "php": ">=7.1",
```

Please use more specific version. As this can easily break in the future. Composer will then install this version when there will be no other version for php 9.0 ;-)
```suggestion
        "php": "^7.1 || ^8.0",
```